spec/openapi.yml

```diff
@@ -209,6 +211,9 @@ paths:
   '/authn/{account}/api_key':
     $ref: 'authentication.yaml#/components/paths/RotateApiKey'
 
+# ========== STATUS ===========
+  '/whoami':
+    $ref: 'status.yml#/components/paths/WhoAmI'
```

Comment:
(nit) It looks like we separate endpoints with an extra newline, so we should add one here as well

Reply:
marked resolved but not fixed